docs/prior-art.md

```diff
@@ -89,3 +89,59 @@ syntax simple. However, it implies that enumerations (such as versions)
 need to be redefined as long lists of flags, along with long lists
 of dependencies and constraints. Ebuilds are bash scripts, so this
 is often done using loops.
+
+## Conda
+
+Conda is a binary-only package ecosystem. It was created by [package
+maintainers who were frustrated with python's rough support for binary
+packages](https://jakevdp.github.io/blog/2016/08/25/conda-myths-and-misconceptions/#Myth-#4:-Creating-conda-in-the-first-place-was-irresponsible-&-divisive).
+The package collection includes both high-level integrations (PyTorch, 
+GDAL, etc.) and low-level dependencies such as compilers and C++ runtimes.
+As such, Conda has approximately the same control over the runtime environment
+as base operating systems. It has proven especially useful for providing an
+up-to-date package collection on older operating systems, where the system
+package manager may not have a current package selection.
```

Comment:
Perhaps worth saying explicitly that everything is installed in a semi-isolated environment, and that you can create multiple different environments. That's IMO the killer feature here.